inc/cachify.class.php

```diff
@@ -641,7 +641,7 @@ public static function add_dashboard_count( $items = array() )
 
 		/* Right now item */
 		$items[] = sprintf(
-			'<a href="%s" class="cachify-icon cachify-icon--%s" title="%s: %s">%s esc_html__( 'Cache', 'cachify' )</a>',
```

Comment:
:+1: 
Also a hint for translators would be finde, the word is to short to understand the context, like 

```php
// Translators: %s is the plugin name.
```